### DISCUSSION TITLE
Hide Roles [in dashboard] depending upon roles

### DISCUSSION THREAD

**Discussion by BilalMajeedDarvis** [pktharindu/nova-permissions#60] [Q&A] [answered]
How can we hide roles depending upon the role/s given to a user? Also how can we hide certain role/s? For instance there is super admin and admin but we don't them in our roles so how can we accomplish that? Thanks

**Comment by pktharindu** [comment, marked as answer]
Hey @BilalMajeedDarvis, as per [Nova docs](https://nova.laravel.com/docs/4.0/resources/authorization.html#index-filtering), you could prevent specific roles from being listed in the resource index by extend the `Role` resource and filtering them out through the `indexQuery()`. 

For the rest of the actions like view, update, delete, etc, it can be done via the `RolePolicy`.

**Comment by sweebee** [comment]
Fc
